projects/json-sanitizer/Dockerfile

```diff
@@ -22,15 +22,14 @@ RUN git clone --depth 1 https://github.com/google/fuzzing
 RUN cat fuzzing/dictionaries/json.dict \
         fuzzing/dictionaries/html.dict \
         fuzzing/dictionaries/xml.dict \
-      > $OUT/DenylistFuzzer.dict
-RUN cat fuzzing/dictionaries/json.dict \
-        fuzzing/dictionaries/html.dict \
-        fuzzing/dictionaries/xml.dict \
-      > $OUT/IdempotenceFuzzer.dict
-RUN cat fuzzing/dictionaries/json.dict \
```

Comment:
@jonathanmetzman Copy & pasting this is where the weird `cp`s came from. Should I fix the nits and ultra nits in follow-up PRs?

Reply:
yes just fix them all in another PR.